sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.4.0-preview.1 (Unreleased)
+## 12.4.0 (2020-08-31)
```

Comment:
should this be 12.3.2?

Reply:
We have an interface change (grouped `ShareFileClient.OpenRead()` optional parameters into `ShareFileOpenReadOptions`), so we have a minor version bump.